spec/consensus/time.md

```diff
@@ -33,20 +37,53 @@ CometBFT implements two algorithms for computing block times:
 - [BFT Time][bft-time]: the algorithm adopted in versions up to `v0.38.x`;
   available, in legacy mode, in version `v1.x`.
 
-- [Proposer-Based Timestamps][pbts-spec]: introduced in version `v1.x`,
+- [Proposer-Based Timestamps (PBTS)][pbts-spec]: introduced in version `v1.x`,
   as a replacement for BFT Time.
 
-Users are strongly encouraged to adopt PBTS in new chains, or to switch to PBTS
+Users are strongly encouraged to adopt PBTS in new chains or switch to PBTS
 when upgrading existing chains.
 
-The table below compares BFT Time and PBTS algorithms in terms of properties:
+### Comparison
 
-| Algorithm | Time Monotonicity | Byzantine Fault Tolerance   | Relation to real time |
-------------|-------------------|-----------------------------|-----------------------|
-| BFT Time  | Guaranteed        | Tolerates `< 1/3` Byzantine nodes     | Best effort and **not** guaranteed |
-| PBTS      | Guaranteed        | Tolerates `< 2/3` Byzantine nodes     | Guaranteed within configured synchronous parameters: `PRECISION` and `MSGDELAY` |
+The table below compares BFT Time and PBTS algorithms in terms of the above enumerated properties:
+
+| Algorithm | Time Monotonicity | Byzantine Fault Tolerance         | Relation to real time                                                                         |
+|-----------|:-----------------:|:---------------------------------:|-----------------------------------------------------------------------------------------------|
+| BFT Time  | Guaranteed        | Tolerates `< 1/3` Byzantine nodes | Best effort and **not guaranteed**.                                                           |
+| PBTS      | Guaranteed        | Tolerates `< 2/3` Byzantine nodes | Guaranteed with `ACCURACY` determined by the consensus parameters `PRECISION` and `MSGDELAY`. |
+
+Note that by Byzantine nodes we consider both malicious nodes, that purposely
+try to increase or decrease block times, and nodes that produce or propose
+inaccurate block times because they rely on inaccurate local clocks.
+
+For more details, refer to the specification of [BFT Time][bft-time] and [Proposer-Based Timestamps][pbts-spec].
+
+## Adopting PBTS
+
+The Proposer-Based Timestamp (PBTS) algorithm is the recommended algorithm for
+producing block times.
+
+As of CometBFT `v1.x`, however, PBTS is not enabled by default, either for new
+chains using default values for genesis parameters, nor for chains upgrading to
+newer CometBFT versions, for backwards compatibility reasons.
+
+Enabling PBTS requires configuring some consensus parameters:
```

Comment:
Please consider adding a link here to a reference for _consensus params_ (either docs.cometbft.com or spec/core/params.md).

Reply:
This file is outdated, but commenting here to not forget to consider.